examples/rbpf_plugin.rs

```diff
@@ -5,8 +5,7 @@
 use std::io::Read;
 
 // Helper function used by https://github.com/Alan-Jowett/bpf_conformance/blob/main/tests/call_unwind_fail.data
-fn _unwind(a: u64, _b: u64, _c: u64, _d: u64, _e: u64) -> u64
-{
+fn _unwind(a: u64, _b: u64, _c: u64, _d: u64, _e: u64) -> u64 {
```

Comment:
You've got a number of changes that are unrelated to the rest of the PR in this file. Let's leave formatting fixes out of this, please.

Reply:
Sorry about the formatting fixes, my autoformatter must have kicked in, I will make sure that the final changes don't touch the code formatting